kisly/todo.js

```diff
@@ -0,0 +1,41 @@
+const logTask = (task) => {
+    taskCounter = 0
+    console.log(task, ":")
+    for (i in todoList.list) {
+        if(todoList.list[i] == task){
+            console.log("  ", i);
+            taskCounter++
+        } 
+    }
+    if(taskCounter == 0) {
+        console.log("-");
+    return 0;
```

Comment:
```suggestion
    noTask = true;
    console.log(task, ":")
    for (i in todoList.list) {
        if(todoList.list[i] == task){
            console.log("  ", i);
            noTask = false;
        } 
    }
    if(noTask) {
        console.log("-");
```
Кажется здесь уместнее использовать булево значение, так как количество задач ни на что не влияет.
Также не увидел где используется `return 0`